tests/pipeline/test_events.py

```diff
@@ -267,6 +267,65 @@ def check_next_event_indexer(self,
                 self.assertEqual(computed_index, -1)
 
 
+class EventsLoaderEmptyTestCase(WithAssetFinder,
+                                WithTradingSessions,
+                                ZiplineTestCase):
+    START_DATE = pd.Timestamp('2014-01-01')
+    END_DATE = pd.Timestamp('2014-01-30')
+
+    @classmethod
+    def init_class_fixtures(cls):
+        cls.raw_events = pd.DataFrame(columns=["sid", "timestamp",
+                                               "event_date", "float", "int",
+                                               "datetime", "string"])
+        cls.next_value_columns = {
+            EventDataSet.next_datetime: 'datetime',
+            EventDataSet.next_event_date: 'event_date',
+            EventDataSet.next_float: 'float',
+            EventDataSet.next_int: 'int',
+            EventDataSet.next_string: 'string',
+            EventDataSet.next_string_custom_missing: 'string'
+        }
+        cls.previous_value_columns = {
+            EventDataSet.previous_datetime: 'datetime',
+            EventDataSet.previous_event_date: 'event_date',
+            EventDataSet.previous_float: 'float',
+            EventDataSet.previous_int: 'int',
+            EventDataSet.previous_string: 'string',
+            EventDataSet.previous_string_custom_missing: 'string'
+        }
+        cls.loader = EventsLoader(
+            cls.raw_events, cls.next_value_columns, cls.previous_value_columns
+        )
+        cls.ASSET_FINDER_EQUITY_SIDS = [0, 1]
+        cls.ASSET_FINDER_EQUITY_SYMBOLS = ['A', 'B']
+        super(EventsLoaderEmptyTestCase, cls).init_class_fixtures()
```

Comment:
So, the other config has been moved to the test method, but the call to super's `init_class_fixtures` does have to be after assigning `ASSET_FINDER_EQUITY_SIDS` and `ASSET_FINDER_EQUITY_SYMBOLS` because those need to be set before we call `WithAssetFinder`'s `init_class_fixtures`.